package.json

```diff
@@ -180,4 +181,4 @@
       ]
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
I assume we'll need some sort of `shield-studies-addon-utils` devDependency, too?